fakerator.d.ts

```diff
@@ -104,7 +103,7 @@ declare module 'fakerator' {
          * @param precision Can generate floating number if you define this parameter. Note: Do not use 0
```

Comment:
> Why do you remove the default value of method parameters?

I can add [default value](https://jsdoc.app/tags-param.html#optional-parameters-and-default-values:~:text=An%20optional%20parameter%20and%20default%20value) as part of the JSDoc documentation so it can be useful for intellisense

```suggestion
         * @param {number} [precision=1] Can generate floating number if you define this parameter. Note: Do not use 0
```